README.md

```diff
@@ -3,7 +3,19 @@
 Discover current Supervisor Services offered to support modern applications through vSphere Services.  
 New service will be added overtime with the goal to continue to empower your DevOps communities.
 
-*Supervisor Services are only available with Supervisor Clusters enabled using NSX-T*
+Supervisor Services **are only** available with Supervisor Clusters enabled **using NSX-T**.
+
+## Supported vSphere Versions
+
+| Supervisor Service | vSphere 7 | vSphere 8 |
+| :--: | :--: | :--: |
+| vSAN Data Persistence Platform Services - MinIO, Cloudian and Dell ObjectScale| ✅ | ✅ |
+| Backup \& Recovery Service - Velero | ✅ | ✅ |
+| Certificate Management Service - Cert-Manager | ✅ | ✅ |
+| Cloud Native Registry Service - Harbor | ❌ | ✅ |
+| Kubernetes Ingress Controller Service - Contour | ❌ | ✅ |
+| Service Discovery Service - External DNS | ✅ | ✅ |
```

Comment:
I'm surprised we call external DNS a "service discovery service"?

Reply:
I think this comes from the official descripion [HERE](https://github.com/kubernetes-sigs/external-dns).
> ExternalDNS makes Kubernetes resources discoverable via public DNS servers.

Alternatives?
- DNS Services
- DNS Automation Service
- DNS Lifecycle-Management Service

Just a couple of ideas.

Reply:
I'd say `Dynamic control of DNS records` instead.

Reply:
As a short description okay but as a Supervisor Service name? Like `Dynamic Control of DNS Records Service`?
Find it a little bit too long, isn't it?